.devcontainer/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM alpine:3.12.0
+
+# Install alp packages
+RUN apk add bash zsh curl git
+
+# Install node
+RUN apk add --update nodejs npm
+
+# Install go
+COPY --from=golang:1.14.4-alpine /usr/local/go/ /usr/local/go/
```

Comment:
Didn't know this is feasible. 

Why not just based on this container from the beginning?
In line 1, 

FROM golang:1.14.4-alpine

Reply:
Yeap, thanks for this. I didn't see the go container for vscode.
changed to this `FROM mcr.microsoft.com/vscode/devcontainers/go:dev-1.15`